### PR TITLE
Ot99 48 endpoint delete news

### PR DESCRIPTION
**Pull request by TomasFogwill**
**Summary**

1. Added NewsRepository
2. Added NewsService interface
3. Added NewsServiceImpl implementation
4. Created endpoint DELETE /news/{id}
5. Created logic in service that do the soft delete and set the delete date
**Evidence**
sql: select*from news;
evidence
evidence1
sql: select*from news; after calling the endpoint
evidence3
calling the enpoint again, we can see that, cause the soft delete, doesnt found a new
evidence2

### CODE REVIEW

**Review by nicolascareri — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+@Service
+public class NewsServiceImpl implements NewsService {
+
+    @Autowired
+    NewsRepository newsRepository;
+
+    @Override
+    public boolean deleteNew(Long id) {
+        if (newsRepository.existsById(id)) {
+           News news = newsRepository.findById(id).get();
```
Comment (nicolascareri) [inline src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java:23]: usar orElseThrow

File: src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.stereotype.Service;
+
+@Service
+public class NewsServiceImpl implements NewsService {
+
+    @Autowired
+    NewsRepository newsRepository;
+
+    @Override
+    public boolean deleteNew(Long id) {
```
Comment (nicolascareri) [inline src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java:21]: retornar void

File: src/main/java/com/alkemy/ong/controller/NewsController.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/news")
+public class NewsController {
+
+    @Autowired
+    NewsService newsService;
+
+    @DeleteMapping("/:{id}")
```
Comment (nicolascareri) [inline src/main/java/com/alkemy/ong/controller/NewsController.java:20]: el : esta demas

**Review by nicolascareri — CHANGES_REQUESTED** [review]

**Comment by TomasFogwill** [comment]
before1
before2
deleting
after1
after2
metodo1
metodo2
metodo3

**Review by nicolascareri — APPROVED** [review]

**Pull request merged**